torch/_inductor/codegen/wrapper.py

```diff
@@ -1570,6 +1568,10 @@ def generate_example_arg_value(self, arg, arg_type=None, raw_arg=None, index=Non
                     fallback=config.unbacked_symint_fallback,
                 )
             )
```

Comment:
Before, for str args like "abc", this will resolve to `config.unbacked_symint_fallback`. 

But now, we'll do `str(arg)` for str args.